packages/devextreme/js/__internal/grids/grid_core/views/m_columns_view.ts

```diff
@@ -841,6 +841,10 @@ export class ColumnsView extends ColumnStateMixin(modules.View) {
 
     this._setCellAriaAttributes($cell, cellOptions);
 
+    if (options.row.node && options.row.node.hasChildren) {
```

Comment:
This code should be included in the method above `_setCellAriaAttributes`. If necessary, pass there an additional argument.